### ISSUE TITLE
Scheduling Notebooks <Data Scientist><Microsoft Fabric>

### ISSUE THREAD

**Issue by aaryan2134** [microsoft/studentambassadors#234]
### Describe the bug

No confirmation on scheduling notebooks

### What happened?

Severity:
Medium

Persona:
Data Scientist

When I tried to schedule a notebook, it got scheduled successfully but the applied button only gets greyed out and there is no confirmation to the user. I had to close the window and verify that it was indeed scheduled.

![image](https://github.com/microsoft/studentambassadors/assets/22559451/43d399c4-aee0-40ca-a5c0-a214a6136c2c)

### What did you expect to happen?

A confirmation message should be shown to the user once the notebook is successfully scheduled

### How would you reproduce the bug?

1. Create a notebook
2. Go to schedule
3. Set the details
4. Click on Apply

### Did this happen via Desktop and/or Smartphone?

Desktop

### Operating System

Windows 10

### What browsers are you seeing the problem on?

Chrome

### What is your current role?

Student Ambassador

### What technical topic is this bug related to?

AI/Data Science/Machine Learning

**Comment by Bullish0407** [comment]
Is this taken seriously?

**Comment by AnthonyByansi** [comment]
Hi @Bullish0407 

The issue has been resolved and is no longer apparent. The Fabric development team has addressed it. Are you still experiencing the same problem on your end?

**Comment by Bullish0407** [comment]
Correct, no confirmation after I scheduled it. Also it didn't work, even thought it would say 'next refresh in x minutes'.
I did a workaround via a pipeline, which made it work.

Overall I am not impressed by my experience of trying to use this. A good point is your quick response, so I am open to give my feedback as a user.
